Resources/public/js/views/services/ez-roleserversideviewservice.js

```diff
@@ -0,0 +1,208 @@
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Role Server Side View Service class.
+     *
+     * @namespace eZ
+     * @class RoleServerSideViewService
+     * @constructor
+     * @extends eZ.ServerSideViewService
+     */
+    Y.eZ.RoleServerSideViewService = Y.Base.create('roleServerSideViewService', Y.eZ.ServerSideViewService, [], {
+        initializer: function () {
+            this.on('*:contentDiscover', function (e) {
+                e.config.contentDiscoveredHandler = Y.bind(this._assignRole, this);
+            });
+        },
+
+        /**
+         * Assign the role to the contents after the user has chosen them in
+         * the universal discovery widget
+         *
+         * @method _assignRole
+         * @protected
+         * @param {EventFacade} e
+         */
+        _assignRole: function (e) {
+            var data = e.target.get('data'),
+                userService = this.get('capi').getUserService(),
+                discoveryService = this.get('capi').getDiscoveryService(),
+                service = this;
+
+            this._assignRoleNotificationStarted(
+                data.roleId, data.roleName, e.selection
+            );
+
+            userService.loadRole(data.roleId, function (error, role) {
+                var tasks = new Y.Parallel(),
+                    end = tasks.add(function (err, response) {
+                        if (err) {
+                            //service._error('tasks.add() failed: ' + err.message);
+                            console.log('tasks.add() failed: ', err.message);
+                            return;
+                        }
+
+                        console.log('end'); // TODO just to let us know that it was picked, error handling should be here
+                    });
+
+                if (error) {
+                    service._error('loadRole failed');
+                    return;
+                }
+
+                Y.Array.each(e.selection, function (struct) {
+                    // TODO assign with limitation
+                    //var limitation = {};
+                    var limitation = { // DUMMY DATA
+                        "_identifier": "Section",
+                        "values": {
+                            "ref": [{
+                                "_href": "/api/ezp/v2/content/sections/1",
+                                "_media-type": "application/vnd.ez.api.Section+json"
+                            }, {
+                                "_href": "/api/ezp/v2/content/sections/4",
+                                "_media-type": "application/vnd.ez.api.Section+json"
+                            }]
+                        }
+                    };
+                    var roleAssignInputStruct = userService.newRoleAssignInputStruct(role, limitation),
+                        contentTypeIdentifier = struct.contentType.get('identifier'),
+                        contentIdStr = struct.content.get('id');
```

Comment:
`id` is full ID and we anly need the number, so maybe changing it to: 
```javascript
contentIdStr = struct.content.get('contentId');
```
It will simplify a bit later tasks